Docs for data model, localization, https redirect (#12876)
metabase/metabase: docs/administration-guide/localization.md

## Localization

The **Localization** settings allow you to set global defaults for your Metabase instance. Localization settings include options for:

- **Language**
- **Date and time**
- **Numbers**
- **Currency**

The **Localization** settings can be found in the **Admin Panel** under the **Settings** tab.

### Instance language

> The default language for all users across the Metabase UI, system emails, pulses, and alerts. If a user would like to use a language different from this default, they can individually override this default language from their own account settings.

Reviewer: ```suggestion
The default language for all users across the Metabase UI, system emails, pulses, and alerts. Users can pick a different language of their preference from their own account settings page.
```
